docs/hub/gguf-llamacpp.md

```diff
@@ -1,16 +1,46 @@
 # GGUF usage with llama.cpp
 
-Llama.cpp directly allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
+Llama.cpp allows you to download and run inference on a GGUF simply by providing a path to the Hugging Face repo path and the file name. llama.cpp would download the model checkpoint in the directory you invoke it from:
 
 ```bash
-./main \
+./llama-cli \
   --hf-repo lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF \
   -m Meta-Llama-3-8B-Instruct-Q8_0.gguf \
   -p "I believe the meaning of life is " -n 128
 ```
 
+Additionally, you can invoke an OpenAI spec chat completions endpoint directly using the llama.cpp server
+
+```bash
+./llama-server \
+  --hf-repo lmstudio-community/Meta-Llama-3-8B-Instruct-GGUF \
+  -m Meta-Llama-3-8B-Instruct-Q8_0.gguf \
+  -p "I believe the meaning of life is " -n 128
+```
+
+After running the server you can simply utilise the endpoint as below:
```

Comment:
Are there guidelines about US vs UK spelling? (re:utilise vs utilize)

Reply:
Alternative: call the endpoint

Reply:
I follow UK spelling 🏰 :p